javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java

```diff
@@ -73,6 +77,14 @@ private void alterCommitIdColumnIfNeeded() {
         }
     }
 
+    /**
+     * This method is needed for upgrading TEXT columns to VARCHAR(MAX) since TEXT is deprecated.
+     */
+    private void alterMssqlTextColumns() {
```

Comment:
this method should be called only once, if col type hasn't been altered yet, see `alterCommitIdColumnIfNeeded()`